.github/workflows/appsec.yml

```diff
@@ -202,27 +202,15 @@ jobs:
       - uses: codecov/codecov-action@v3
 
   next:
-    strategy:
-      matrix:
-        node-version: [16]
-        range: ['>=9.5 <11.1', '>=11.1 <13.2']
-        include:
-          - node-version: 18
-            range: '>=13.2'
     runs-on: ubuntu-latest
     env:
       PLUGINS: next
-      RANGE: ${{ matrix.range }}
     steps:
       - uses: actions/checkout@v4
-      - uses: ./.github/actions/testagent/start
```

Comment:
Was deleting the testagent parts intentional?

Reply:
No it wasn't repushed